README.md

```diff
@@ -1,4 +1,4 @@
-# frontend-app-program-manager
+# frontend-app-program-console
```

Comment:
suggested description (as that also seems to be incorrect):
` A micro-frontend for administering edX program membership through the Registrar API. `